### ISSUE TITLE
Make page title of html report configurable

### ISSUE THREAD

**Issue by hstaudacher** [siom79/japicmp#84]
It would be very nice if the title of a report could be configured with a parameter in the maven plugin.

**Comment by siom79** [comment]
Do you mean with "title" only the `<title'/>` element of the HTML report or also the string `JApiCmp-Report` at the top of the page?

**Comment by hstaudacher** [comment]
both ;)

**Comment by siom79** [comment]
I have committed a corresponding change set on the `develop` branch.

If you like, you can try it out:
```
git clone https://github.com/siom79/japicmp.git
cd japicmp
mvn install
```
After updating to version 0.6.1-SNAPSHOT you can try:
```
<parameter>
    ...
    <htmlTitle>Title of report</htmlTitle>
</parameter>
```

**Comment by hstaudacher** [comment]
Works like a charm. Thank you!

**Comment by hstaudacher** [comment]
when do you plan to release 0.6.1?

**Comment by siom79** [comment]
Released with version 0.6.1.
